pib/tfstatereader.py

```diff
@@ -66,12 +70,12 @@ def render(self):
             name=name, resource_name=self.resource_name, data=self.config)
 
 
-def create_aws_elasticsearch_domain(tf_data):
+def _create_aws_elasticsearch_domain(tf_data):
     return {'HOST': tf_data['attributes']['endpoint'],
             'PORT': "80"}  # doesn't have one...
```

Comment:
This probably should be whatever the actual default port for ES HTTP API is. I don't think it's 80